app/steps/ui/deceased/alias/template.html

```diff
@@ -0,0 +1,23 @@
+{% extends "includes/two_thirds_form.html" %}
+
+{% from "widgets/textfields.html" import textField %}
+{% from "widgets/radiobuttons.html" import optionsRadio %}
+
+{% block question %}
+    <h1 class="form-title heading-large">{{content.question | replace('{deceasedName}', fields.deceasedName.value) | safe }}</h1>
```

Comment:
Change this accordingly if you amend the method to replace the deceased name.

Reply:
agreed with David to leave template substitution as PA journey